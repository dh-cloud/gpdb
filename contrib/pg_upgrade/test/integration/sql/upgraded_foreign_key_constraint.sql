SELECT * FROM mfk;
SELECT * FROM pt;
SELECT * FROM pt_another;
SELECT * FROM non_pt;
