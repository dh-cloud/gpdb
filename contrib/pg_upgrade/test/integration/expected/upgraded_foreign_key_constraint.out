SELECT * FROM mfk;
 a 
---
 1 
 2 
(2 rows)
SELECT * FROM pt;
 a 
---
 1 
 2 
(2 rows)
SELECT * FROM pt_another;
 a 
---
 1 
 2 
(2 rows)
SELECT * FROM non_pt;
 a 
---
 1 
 2 
(2 rows)
